packages/nvidia-k8s-device-plugin/nvidia-k8s-device-plugin.service

```diff
@@ -1,7 +1,5 @@
 [Unit]
 Description=Start NVIDIA kubernetes device plugin
-RefuseManualStart=true
-RefuseManualStop=true
```

Comment:
Are we now manually starting/stopping this?

Reply:
> Are we now manually starting/stopping this?

We need to be able to restart it when settings change.

Reply:
Ok, that makes sense, I just wanted to make sure that was the right conclusion I made when reading this.